proconfig/config/base.py

```diff
@@ -7,6 +7,7 @@
         'level': 'DEBUG'
     },
     'service': {
-        'port': os.getenv('PORT')
+        'port': os.getenv('PORT'),
+        'name': 'GFW PRO Text Configuration'
```

Comment:
nitpicking, but the service is not called "gfw pro", it's called "gs pro". I know this is small as hell, but it is used to create logs in CloudWatch, and I can easily see us-in-6-months getting confused as to why we have to logs for "gs pro" there.

Reply:
Good catch, thanks.